scripts/travis-ci/run-test.sh

```diff
@@ -14,12 +14,12 @@ case "$1" in
     8.3.x)
         ln -s $MODULE_DIR $DRUPAL_DIR/modules/og
         cd $DRUPAL_DIR
-        ./vendor/bin/phpunit -c ./core/phpunit.xml.dist $MODULE_DIR/tests
+        ./vendor/bin/phpunit -c ./core/phpunit.xml.dist $MODULE_DIR/
         exit $?
         ;;
     *)
         ln -s $MODULE_DIR $DRUPAL_DIR/modules/og
         cd $DRUPAL_DIR
-        ./vendor/bin/phpunit -c ./core/phpunit.xml.dist --exclude-group=console $MODULE_DIR/tests
```

Comment:
@amitaibu, It seems that OG UI tests never ran

Reply:
Dammit! The existing test that never ran fails :)

Reply:
Oh my, it seems we've opened a pandora case! 😜

Reply:
@amitaibu so my patch to `scripts/travis-ci/run-test.sh` runs also other tests. Something is not OK.